### ISSUE TITLE
Queation about 'invoice company'

### ISSUE THREAD

**Issue by yangcen6** [allegro/allegro-api#10077]
When a buyer wants to issue a corporate invoice, he needs to fill in the company's tax information (as shown in Figure 1). So is `invoice.company` （in Figure 2）in the **/order/checkout-forms/{id}** the company's tax information that the user filled in in Figure 1? What other fields can let us know that the buyer wants to issue a corporate invoice?
 **Figure 1**
![{76BC75E3-3C7C-4E81-AD20-BA3033353AF7}](https://github.com/user-attachments/assets/3761f49e-0e66-4830-8e48-9a0d442647ac)
**Figure 2**
![{76BC75E3-3C7C-4E81-AD20-BA3033353AF7}](https://github.com/user-attachments/assets/79b4426b-02da-49e2-9b16-a37323ce0afe)

**Comment by AureliuszBrussy** [comment]
You are right, you will receive all the information on the request for an invoice in the invoice object. There is no other data structure where we return such information.
